NuRadioReco/detector/detector_base.py

```diff
@@ -208,6 +208,38 @@ def __init__(self, source='json', json_filename='ARIANNA/arianna_detector_db.jso
             logger.info("the correct antenna model will be determined automatically based on the depth of the antenna")
         self._antenna_by_depth = antenna_by_depth
 
+    @property
+    def assume_inf(self):
+        """
+        Getter function for the `assume_inf` attribute
+        """
+        return self.__assume_inf
+
+    @assume_inf.setter
+    def assume_inf(self, value):
```

Comment:
should that function than not raise a warning if the value is not set?

Reply:
Are you talking about the getter or the setter?

Reply:
setter

Reply:
Why would a warning be necessary? In any case, these values are set in the `__init__` so I don't see how the value could ever not be set.

Reply:
It might be over doing it